source/index.txt

```diff
@@ -22,59 +22,67 @@ MongoDB Node Driver
 Introduction
 ------------
 The official MongoDB Node.js driver allows Node.js applications to
-connect to MongoDB and work with data. 
+connect to MongoDB and work with data.
 
 .. note::
-   
+
    These docs are for version 3.5 of the MongoDB Node.js driver. If
    you're looking for an older version of the MongoDB Node.js driver
-   docs, `see the Legacy Node.js driver documentation
-   <https://mongodb.github.io/node-mongodb-native/>`_. For the main
-   MongoDB documentation, see the :manual:`MongoDB Manual </>`.
+   docs, :node-docs:`see the Legacy Node.js driver documentation </>`.
+   For the main MongoDB documentation, see the :manual:`MongoDB Manual
+   </>`.
 
 Quick Start
 -----------
-Jump right in with the `Quick Start </quick-start/>`_ to learn how to
-establish a connection to MongoDB Atlas and begin to work with data
+Jump right in with the :doc:`Quick Start </quick-start>`
```

Comment:
Should have mentioned this last review, but we should avoid idioms; this one is also not used completely correctly. It should be "Jump right in to learn..." as the "with" preposition would imply something that you're jumping in with, e.g. "Jump right in with your learning cap to learn...". If you think Nathan really wanted that idiom, it could be reworded: 
> Jump right in to learn... MongoDB Atlas and work with data in the :doc:`Quick Start </quick-start>` section.